doc/riscv-bfloat16-appx-rationale.adoc

```diff
@@ -0,0 +1,67 @@
+[appendix]
+[[BFloat16_appx_rationale]]
+= Extension Rationale
+
+== Format Rationale
+Various choices were made in the RISC-V BFloat16 format and behavior.
+Some of these choices are allowed by IEEE-754 and others are deviations
+from the standard
+
+=== Rounding Modes
+
+==== Round to odd
+Round to odd is not a '754 supported rounding mode. However, it avoids double
+rounding that can occur when accumulating a result in a wider format and then
+converting the result to a narrower format before subsequent usage.
+
+==== Round to nearest - even
+Round to nearest, ties to even is the default '754 rounding format. It is unbiased
+and minimize rounding error.
+
+=== Subnormal Handling
+
+=== NaN handling
+
+=== Zeros and Infinities
+
+== Instruction Rationale
+
+This section contains various rationale, design notes and usage
+recommendations for the instructions in the BFloat16 extension.
+It also tries to record how the designs of instructions were
+derived, or where they were contributed from.
+
+=== Conversion Instructions
+
+
+The most common and important conversion instructions are between BFloat16 and FP32
+(Single Precision).
+
+We chose not to have direct conversion between BFloa16 and other formats as they
+can typically be performed by a combination of instructions. 

```

Comment:
Do we intend to list (and check) those combinations of instructions in the actual specification ?

Reply:
We intend to check that it is possible to move between these other formats. This would likely be in an appendix rather than as a part of the specification proper.